LearningMissionSimulation/PlaygroundMher.cs

```diff
@@ -6,57 +6,42 @@ namespace LearningMissionSimulation
 {
     public class PlaygroundMher
     {
-        public PlaygroundMher()
-        {
-            
-        }
-       
-        //Account account = ObjectGenerator.GenerateAccount();
+        Queue<Account> accountQueue = new Queue<Account>();
 
-        List<Account> accountList = new List<Account>();
+        Dictionary<Guid, Account> accountDictionary = new Dictionary<Guid, Account>();
 
-        Queue<Status> statusQueue = new Queue<Status>();
-
-        // Simulate account creation  process
         public void CreateAccounts(int accountCount)
         {
             int i = 0;
             while (i < accountCount)
             {
-                accountList.Add(ObjectGenerator.GenerateAccount());
-                i++;
-            }
-            i = 0;
-            foreach (var account in accountList)
-            {
-                if (account.Status == Status.Pending)
+                Account account = ObjectGenerator.GenerateAccount();
+                accountDictionary.Add(account.Id, account);
+                if (account.Role == Role.Student || account.Role == Role.Instructor)
                 {
-                    statusQueue.Enqueue(account.Status);
+                    accountQueue.Enqueue(account);
                 }
-                Console.WriteLine("=============== Create Account {0} =================\n", i);
-                Console.WriteLine(account + "\n");
+                Console.WriteLine("========== Create  account {0} =======\n{1}", i, account);
+                Console.WriteLine("\n");
                 i++;
             }
         }
 
-        // Simulate account activati process
         public void ActivateAccounts()
         {
-            int i = 0;
-            foreach (var account in accountList)
+            foreach (var accountStatus in accountQueue)
```

Comment:
Also I would like to see queue-specific methods: how you retrieve the first in the line account and activate it. I don't want to keep accounts in the queue after activation.